simul/platform/mininet.go

```diff
@@ -248,11 +286,17 @@ func (m *MiniNet) Deploy(rc *RunConfig) error {
 	}
 
 	// Copy our script
-	err = app.Copy(m.deployDir, m.mininetDir+"/start.py")
+	err = app.Copy(m.deployDir, path.Join(m.mininetDir, "start.py"))
 	if err != nil {
 		log.Error(err)
 		return xerrors.Errorf("copying: %v", err)
 	}
+	// As the `start.py` is kept in the go-pkg directory now,
```

Comment:
The go pkg dir should really not be touched at all. If they include the permissions in the source hashes, then this change would cause "go mod verify" to start failing, etc. Safer would be copy to a https://golang.org/pkg/io/ioutil/#TempFile, chmod, and defer a delete to clean it up.

But: if this works, and you're happy with it, we can go with it, and change it later if needed.

Reply:
This doesn't touch the go mod cache dir at all. The file from the go mod cache dir is copied to the `m.deployDir`, which is `$(pwd)/deploy`, and only `$(pwd)/deploy/start.py` is changed.

Of course I would _never_ dare touching the go mod cache dir (cough, cough)...